workspaces/ui/src/utilities/debug-dump.js

```diff
@@ -0,0 +1,33 @@
+import * as deepCopy from 'deepcopy';
+
+export function debugDump(specService, captureId) {
+  return async function () {
+    const events = await specService.listEvents();
+    const session = await specService.listCapturedSamples(captureId);
+    const sessionCleaned = deepCopy(session);
```

Comment:
I think we would be fine with a `const copy = JSON.parse(JSON.stringify(x))` here instead of adding a dependency